README.md

```diff
@@ -0,0 +1,193 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div
+
```

Comment:
I would recommend removing the Logo from your README file. Optionally you can replace it with your own logo if you have one.

Reply:
murple logo removed